docs/development/authentication.md

```diff
@@ -42,6 +42,25 @@ details worth understanding:
   password in the development environment, those passwords won't
   work.  It also prints out the user's **current** API key.
 
+### Apple
+
+* Visit https://developer.apple.com/account/resources/,
+  [Create a Sign In Certificate](https://help.apple.com/developer-account/?lang=en#/dev77c875b7e)
+
+* [Create a Services ID](https://help.apple.com/developer-account/?lang=en#/dev1c0e25352). Fill the redirect URI with
```

Comment:
URL, not URI.

[The standard term is "URL"](https://chat.zulip.org/#narrow/stream/92-learning/topic/URLs/near/794826). Sometimes an existing API says "URI" and it makes sense to match it; but here, all the Apple docs say "URL" too.

This also applies to the prod instructions.

Reply:
Ohh, Thanks for pointing to czo topic